PostProcessing/Editor/PostProcessLayerEditor.cs

```diff
@@ -203,6 +207,22 @@ void DoFog(Camera camera)
             EditorGUILayout.Space();
         }
 
+        void DoDepthOfField(Camera camera)
```

Comment:
I'm not too fond of having this on `PostProcessLayer`... It looks out of place IMO.

The way you've done it means that you can't unlock DoF for special/artsy shots, DoF settings on volumes are ignored which makes the UX a bit confusing ("I'm changing DoF settings on a high priority volume and it doesn't do anything"). Optimally the physical camera would have a "priority" setting the same way volumes do, so that you'd be able to quickly switch between physcam controls and absolute controls using a volume.

I believe this "physcam priority" could live on the `PostProcessLayer` itself (no need to put it straight into the physcam). From there, if physcam is enabled the layer would simply create a dummy hidden volume with that same priority value and hold its custom DoF settings. From what I can see, the only setting in DoF that would need to be exposed to make this work is the sensor height (film size).